incubator/kube-aws-ingress-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "kube-aws-ingress-controller.name" . }}
+  namespace: kube-system
+  labels:
+    app: {{ template "kube-aws-ingress-controller.name" . }}
+    chart: {{ template "kube-aws-ingress-controller.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: {{ template "kube-aws-ingress-controller.name" . }}
+      component: ingress
+  template:
+    metadata:
+      labels:
+        application: {{ template "kube-aws-ingress-controller.name" . }}
```

Comment:
Add labels `app` and `release`. Label `application` should be removed.

Reply:
Same, done. Thanks